.github/workflows/build_training.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   build:
+
```

Comment:
interesting change :D

Reply:
I wanted all three files to have the same structure :P